dumper/app/src/main/resources/oracle-stats/dba/native/db-info.sql

```diff
@@ -0,0 +1,23 @@
+-- Copyright 2022-2024 Google LLC
+-- Copyright 2013-2021 CompilerWorks
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--     http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+SELECT
+  A.dbid "DbId",
+  A.name "Name",
+  A.db_unique_name "DbUniqueName",
+  NULL "ConId",
+  NULL "ConDbId",
+  NULL "Cdb",
+  A.database_role "DatabaseRole"
+FROM v$database A
```

Comment:
v$database table is the same for CDB and non-CDB so we could keep the single SQL definition. Do you expect any issues with extracting this data from non-CDB instance or Oracle 11 (version before multi tenant was introduced).

Reply:
https://docs.oracle.com/cd/E11882_01/server.112/e40402/dynviews_1097.htm#REFRN30047 looks like in oracle 11 there is no `ConId`, `ConDbId`, `Cdb` columns so we need to keep this version to support older oracle version.

Reply:
True, I was using the same source